src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -745,7 +745,10 @@ export class CalciteCombobox {
             "wrapper--active": active,
             "wrapper--single": single
           }}
-          onClick={() => this.setFocus()}
+          onClick={(e) => {
+            e.stopPropagation();
```

Comment:
I believe [this is the issue](https://github.com/Esri/calcite-components/pull/995) here. Could you try updating [`setInactiveIfNotContained`](https://github.com/Esri/calcite-components/blob/7201c26fdae3a8c15e898e6bdda26c80ba0611f4/src/components/calcite-combobox/calcite-combobox.tsx#L305) to use `composedPath` instead? 

In general, I don't think we should stop propagation if the event is not strictly internal.

Reply:
will composed path fall back gracefully in ie11 when it's using the pseudo-shadow dom?

Reply:
It's polyfilled by the compiler, but can't say I've tested it in IE11.